src/lib/spinel/spinel.h

```diff
@@ -4836,6 +4836,38 @@ enum
      */
     SPINEL_PROP_DNSSD_HOST = SPINEL_PROP_DNSSD__BEGIN + 3,
 
+    /// DNS-SD Service
+    /**
+     * Format `UUUSA(U)SdLD`: Inserted/Removed
+     *
+     * `U`    : The host name (does not include domain name).
+     * `U`    : The service instance name label (not the full name).
+     * `U`    : The service type (e.g., "_mt._udp", does not include domain name).
+     * `S`    : The count of sub-type labels.
+     * `A(U)` : Array of sub-type labels (can be NULL if no label).
+     * `S`    : The service port number.
+     * `d`    : Encoded TXT data bytes.
```

Comment:
Should we also include 
```
    uint16_t           mPriority;            ///< The service priority.
    uint16_t           mWeight;              ///< The service weight.
    uint32_t           mTtl;                 ///< The service TTL in seconds.
```

Reply:
Currently it seems these fields are not used by the mDNS Publisher on the host. But let's add them in case these they are needed in the future :) 

Also added `mClass` and `mTtl` for KeyRecord.